src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,25 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.FruitServiceImpl;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
 public class HelloWorld {
```

Comment:
we can also rename this class to something like FruitShop, but that is not important